Havit.Blazor.Documentation/Pages/Components/HxGridDoc/HxGrid_Documentation.razor

```diff
@@ -70,6 +70,13 @@
 		</p>
 
 
+        <DocHeading Title="Pagination display" />
+        <p>
+            <code>HxGrid</code> uses the <code>HxPager</code> to display pagination, unless a value for the <code>PaginationTemplate</code> parameter is provided.
+            Context information is provided via <code>HxPaginationContext</code> to your pager.
+        </p>
```

Comment:
Please provide a full demo showcasing the feature `HxGrid_Demo_CustomPagination.razor`.